django/api/viewsets/application_form.py

```diff
@@ -41,3 +41,22 @@ def get_serializer_class(self):
             return self.serializer_classes.get(self.action)
 
         return self.serializer_classes.get("default")
+
+    @action(detail=False, methods=["GET"], url_path="check_status")
+    def check_status(self, request, pk=None):
+        drivers_license = request.query_params.get("drivers_license", None)
+        application_status = GoElectricRebateApplication.objects.filter(
+            drivers_licence=drivers_license
+        ).values_list("status", flat=True)
+
+        if application_status:
+            if application_status[0] in [
```

Comment:
This should compare via the status enum type

ex. `GoElectricRebateApplication.Status.SUBMITTED`

Reply:
This only needs a true/false sent back to the front end. It should be true for these statues:

Initiated
Submitted
Verified
Approved
Redeemed

and false otherwise.